application/console/migrations/m181207_144943_create_invite_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use yii\db\Migration;
+
+/**
+ * Handles the creation of table `invite`.
+ */
+class m181207_144943_create_invite_table extends Migration
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function safeUp()
+    {
+        $this->createTable(
+            'invite',
+            [
+                'id' => 'pk',
+                'user_id' => 'int(11) not null',
+                'uuid' => 'char(36) CHARACTER SET ascii COLLATE ascii_general_ci not null',
```

Comment:
Does the ascii thing matter?

Reply:
Probably not. I was thinking that it would reduce storage space, but I recognize it's trivial.